workspaces/ui-v2/src/spectacle-implementations/cloud-viewer.tsx

```diff
@@ -0,0 +1,163 @@
+import * as React from 'react';
```

Comment:
This is basically copy-pasted from `spectacle-implementations/local.tsx`, but I swapped out the spec fetching code. Would be good to generalize this, but I don't want to conflict with ongoing work. @devdoshi might be worth talking about the right way to generalize this?

Reply:
yep once we have the next few we'll see how they converge/diverge